packages/workspace/src/command-line/list.ts

```diff
@@ -41,8 +42,17 @@ async function listHandler(args: YargsListArgs) {
   if (args.plugin) {
     listPluginCapabilities(args.plugin);
   } else {
-    const corePlugins = await fetchCorePlugins();
-    const communityPlugins = await fetchCommunityPlugins();
+    const corePlugins = fetchCorePlugins();
+    let communityPlugins: CommunityPlugin[];
+    try {
+      communityPlugins = await fetchCommunityPlugins();
+    } catch {
+      output.warn({
+        title: `Community plugins:`,
+        bodyLines: [`Error fetching plugins.`],
+      });
+    }
+
```

Comment:
This can be simplified (no `try`/`catch` and no `let`) to

```suggestion
    const communityPlugins = await fetchCommunityPlugins()
      .catch(() => {
        output.warn({
          title: `Community plugins:`,
          bodyLines: [`Error fetching plugins.`],
        });
        
        return [];
      });
```